editor/find_in_files.cpp

```diff
@@ -1178,3 +1195,179 @@ void FindInFilesPanel::_bind_methods() {
 
 	ADD_SIGNAL(MethodInfo(SIGNAL_CLOSE_BUTTON_CLICKED));
 }
+
+//-----------------------------------------------------------------------------
+
+FindInFilesTab::FindInFilesTab() {
+	add_theme_constant_override("margin_left", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanelDebuggerOverride"), EditorStringName(EditorStyles))->get_margin(SIDE_LEFT));
+	add_theme_constant_override("margin_right", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanelDebuggerOverride"), EditorStringName(EditorStyles))->get_margin(SIDE_RIGHT));
+	add_theme_constant_override("margin_bottom", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanelDebuggerOverride"), EditorStringName(EditorStyles))->get_margin(SIDE_BOTTOM));
```

Comment:
```suggestion
	add_theme_constant_override("margin_top", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles))->get_margin(SIDE_TOP));
	add_theme_constant_override("margin_left", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles))->get_margin(SIDE_LEFT));
	add_theme_constant_override("margin_right", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles))->get_margin(SIDE_RIGHT));
	add_theme_constant_override("margin_bottom", -EditorNode::get_singleton()->get_editor_theme()->get_stylebox(SNAME("BottomPanel"), EditorStringName(EditorStyles))->get_margin(SIDE_BOTTOM));
```
This also needs to be added to `NOTIFICATION_THEME_CHANGED` so when the theme changes, the margins are able to update

Reply:
Yes. BTW, SNAME("BottomPanelDebuggerOverride") should be changed to SNAME("BottomPanel")?